plugins/services/src/js/components/ServiceConfigDisplay.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+
+import ConfigurationMap from '../../../../../src/js/components/ConfigurationMap';
+import ConfigurationMapHeading from '../../../../../src/js/components/ConfigurationMapHeading';
+import ConfigurationMapLabel from '../../../../../src/js/components/ConfigurationMapLabel';
+import ConfigurationMapRow from '../../../../../src/js/components/ConfigurationMapRow';
+import ConfigurationMapSection from '../../../../../src/js/components/ConfigurationMapSection';
+import ConfigurationMapValue from '../../../../../src/js/components/ConfigurationMapValue';
+import ServiceConfigDisplayUtil from '../utils/ServiceConfigDisplayUtil';
+import Util from '../../../../../src/js/utils/Util';
+
+const shouldExcludeItem = (row, appDefinition) => {
+  switch (row.key) {
+    case 'gpus':
+      return appDefinition.gpus === 0;
+    case 'container.volumes':
+      return appDefinition.container == null
+        || appDefinition.container.volumes == null
+        || appDefinition.container.volumes.length === 0;
```

Comment:
We also need to check for a `null` value, which looks like this:

```javascript
return Util.findNestedPropertyInObject(
    appConfig,
    'container.volumes'
  ) == null
  || Util.findNestedPropertyInObject(
    appConfig, 'container.volumes.length'
  ) === 0;
```

So I actually think it looks a little cleaner to do the triple checks:
```javascript
return appConfig.container == null
  || appConfig.container.volumes == null
  || appConfig.container.volumes.length === 0;
```

What do you think?